app/views/user_mailer/welcome_email.html.haml

```diff
@@ -0,0 +1,14 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
+  %body
+    %h1
+      Welcome to example.com, #{@user.email}
+    %p
+      You have successfully signed up to example.com,
+      your username is: #{@user.email}.
+      %br
+    %p
+      To login to the site, just follow this link: #{@url}.
```

Comment:
you can do #{root_url} and then it automatically should know what the website url is as long as you've set it up according to the rails doc [here](http://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views)